production/helm/loki/templates/runtime-configmap.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "loki.fullname" . }}-runtime
+  labels:
+    {{- include "loki.labels" . | nindent 4 }}
+data:
+  runtime.yaml: |
```

If we decide to apply my suggestion about placing the runtime config at `/etc/loki/runtime-config/runtime-config.yaml`, this line should be: 
```suggestion
  runtime-config.yaml: |
```

---

done :)